Dockerfile

```diff
@@ -1,18 +1,53 @@
-FROM ubuntu:bionic
+# syntax=docker/dockerfile:experimental
 
-RUN apt-get update && apt-get install --yes nginx
+# Build stage: Install ruby dependencies
+# ===
+FROM ruby:2.7 AS build-site
```

Comment:
We are not using yet version 2.7 of ruby. The dev image runs with ruby coming from apt (`apt install ruby`) that installs ruby 2.5.

The way to see that is the error that we get when building the docker image with this version of ruby:

``` > [build-site 4/5] RUN bundle install:
#16 0.800 /usr/local/lib/ruby/2.7.0/rubygems.rb:275:in `find_spec_for_exe': Could not find 'bundler' (1.16.1) required by your /srv/Gemfile.lock. (Gem::GemNotFoundException)
#16 0.800 To update to the latest version installed on your system, run `bundle update --bundler`.
#16 0.800 To install the missing version, run `gem install bundler:1.16.1`
#16 0.803       from /usr/local/lib/ruby/2.7.0/rubygems.rb:294:in `activate_bin_path'
#16 0.803       from /usr/local/bin/bundle:23:in `<main>'
------
failed to solve with frontend dockerfile.v0: failed to solve with frontend gateway.v0: rpc error: code = Unknown desc = failed to build LLB: executor failed running [/bin/sh -c bundle install]: runc did not terminate sucessfully
```

This means that the bundler installed is not the same version the linked in the `Gemfile.lock`.

Making the upgrade implies quite a large amount of work (installing from new sources, updating the run script, updating the run script in this project and updating the Gemfile.lock of this project).

I suggest that we keep `FROM ruby:2.5 AS build-site` for now. And think later how to make the ugprade (via the dotrun snap maybe?)